airbyte-webapp/src/components/LabeledControl/ControlLabels.tsx

```diff
@@ -1,26 +1,27 @@
+import className from "classnames";
 import React from "react";
-import styled from "styled-components";
+import { FormattedMessage } from "react-intl";
 
+import { Text } from "components/base/Text";
+import { InfoTooltip } from "components/base/Tooltip";
 import Label from "components/Label";
 
+import styles from "./ControlLabels.module.scss";
+
 export interface ControlLabelsProps {
   className?: string;
   error?: boolean;
   success?: boolean;
   nextLine?: boolean;
   message?: React.ReactNode;
-  errorMessage?: React.ReactNode;
   labelAdditionLength?: number;
   label?: React.ReactNode;
+  infoMessage?: React.ReactNode;
```

Comment:
`infoMessage` was meant to contain specifically the content that should go in an InfoTooltip that is added to the label, if this prop is set. `message`, on the other hand, is the element that is displayed inline after the label itself. This PR moves a lot of the content that was in `message` into `infoMessage`, since that is part of the redesign, but other places in the app still use that `message` field.

Based on your comment, I've renamed this prop to `infoTooltipContent` which hopefully makes that a bit more clear.